.azure/publish-coverage.yml

```diff
@@ -0,0 +1,49 @@
+#
```

Comment:
I avoided this name because it actually has a special meaning for codecov as a [settings file](https://docs.codecov.io/docs/codecov-yaml) so if it gets uploaded to codecov itself by accident the codecov analysis will fail. If that risk is not important to you I will change it.

Reply:
Given that it's in the `.azure` directory, changes of that happening seem non-existent.

Reply:
It'll do for now, let's get CI setup first...